src/main/kotlin/frc/robot/subsystems/shooter/turret/Turret.kt

```diff
@@ -0,0 +1,31 @@
+package frc.robot.subsystems.shooter.turret
+
+import com.ctre.phoenix6.controls.MotionMagicTorqueCurrentFOC
+import edu.wpi.first.units.Units
+import edu.wpi.first.units.measure.Angle
+import edu.wpi.first.wpilibj.DigitalInput
+import edu.wpi.first.wpilibj2.command.SubsystemBase
+import edu.wpi.first.wpilibj2.command.button.Trigger
+import frc.robot.lib.universal_motor.UniversalTalonFX
+import org.littletonrobotics.junction.Logger
+
+class Turret : SubsystemBase() {
+    val motor = UniversalTalonFX(MOTOR_PORT, config = MOTOR_CONFIG)
+    var angleSetpoint = Units.Rotations.zero()
+    val sensor = DigitalInput(SENSOR_PORT)
+    val motionMagicTorque = MotionMagicTorqueCurrentFOC(0.0)
+    val isAtResetPoint =
+        Trigger { sensor.get() }
+            .onTrue(runOnce { motor.reset(Units.Rotations.zero()) })
+    val isAtSetpoint = Trigger { motor.inputs.position == angleSetpoint }
+    fun setAngle(position: Angle) = runOnce {
+        motor.setControl(motionMagicTorque.withPosition(position))
+    }
+
+    override fun periodic() {
+        motor.updateInputs()
+        Logger.processInputs("Subsystem/$name", motor.inputs)
```

Comment:
```suggestion
        Logger.processInputs("Subsystems/$name", motor.inputs)
```
Change to plural to match other subsystem. See [this comment](https://github.com/Galaxia5987/offseason-2025-rapid-react/pull/38#discussion_r2223832122).

Reply:
0671f0fe9e7c0b74765786ed91b4428ea99a3cf5